packages/helium-sub-daos-sdk/src/pdas.ts

```diff
@@ -35,3 +35,14 @@ export function subDaoKey(
     programId
   );
 }
+
+export function stakePositionKey(
+  voterAuthority: PublicKey,
+  depositEntryIdx: number,
+  programId: PublicKey = PROGRAM_ID
+): [PublicKey, number] {
+  return PublicKey.findProgramAddressSync([
+    Buffer.from("stake_position", "utf-8"), voterAuthority.toBuffer(), (new BN(depositEntryIdx)).toBuffer()], 
```

Comment:
Not sure I love the new BN approach here. Shouldn't this be an explicit number of bytes, and the endianess specified?

Reply:
The index is only 1 byte, so endianess doesn't matter. I'll take a look and see if there's a better way to write it

Reply:
const b = Buffer.alloc(1)
b.writeSomethingLE(number)

b